.buildkite/steps/deploy-test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -euo pipefail
+
+wait-for-proxy()
+{
+  PROXY_URL="$1"
+
+  for i in {1..40}; do
+      if curl -s --header "Content-Type: application/json" --data '{"method":"eth_blockNumber","params":[],"id":93,"jsonrpc":"2.0"}' $PROXY_URL > /dev/null;
+      then
+        echo `date +%H:%M:%S`" proxy is available"
+        return 0
+      fi
+      echo `date +%H:%M:%S`" proxy is unavailable - sleeping"
+      sleep 15
+  done
+
+  echo `date +%H:%M:%S`" proxy is unavailable - time is over"
+  return 8938
+}
+
+while getopts t: option; do
+case "${option}" in
+    t) IMAGETAG=${OPTARG};;
+    *) echo "Usage: $0 [OPTIONS]. Where OPTIONS can be:"
+       echo "    -t <IMAGETAG>  tag for cybercoredev/uniswap-v2-core Docker-image"
+       exit 1;;
+esac
+done
+
+export REVISION=latest
+docker run --rm --entrypoint cat cybercoredev/proxy:latest proxy/docker-compose-test.yml > node-and-proxy.yml
+
+docker-compose -f node-and-proxy.yml up -d
+
+function cleanup_docker {
+    echo "Cleanup docker-compose..."
+    docker-compose -f node-and-proxy.yml down
+    echo "Cleanup docker-compose done."
+}
+trap cleanup_docker EXIT
+sleep 10
+
+export PROXY_URL=http://127.0.0.1:9090/solana
+
+echo "Wait proxy..." && wait-for-proxy "$PROXY_URL"
+
+./deploy-test.sh
```

Comment:
You should run deploy-test.sh from container (like in solana-program-library/./buildkite/steps/deploy-tesst.sh).